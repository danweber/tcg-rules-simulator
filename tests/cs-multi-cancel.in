VERSION:2
# [Main] Delete all of your opponent's Monsters with the lowest level.

# p1 has three Lv.4, and four Lv.5, and three of those Lv.5 are phoenix.
# Some have a cancel, some don't.
FIELD1:Four1,CS1-06,PLUG,CS3-Fairy Four2,CS1-06,PLUG,CS3-Fairy Four3,CS1-07,PLUG,CS3-Fairy Five1,CS1-Phoenix
EGGS2:CS2-01
HAND2:CS2-Quick CS2-Quick
DECK1:CS1-02
GAME:P2 T3 HATCHING 9
CMD:HATCH
DUMP:EGGZONE,TRASH

ECHO: -- Use first quick slash, kill the level 4s
CMD:PLAY CS2-Quick

STEP:AUTOANSWER:CAPTURE
DUMP:TRASH

EXIT
