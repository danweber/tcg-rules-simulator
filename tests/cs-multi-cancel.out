FIELD1:Four1,CS1-Sneakycat,PLUG,CS3-Fairyfeatherlink Four2,CS1-Sneakycat,PLUG,CS3-Fairyfeatherlink Four3,CS1-Slipperyshield,PLUG,CS3-Fairyfeatherlink Five1,CS1-Phoenixdefender
FIELD2:
TRASH1:
TRASH2:
EGGZONE1:
EGGZONE2:Xxx,CS2-Doggiedagger
GAME:P2 T3 MAIN 9
ECHO: -- Use first quick slash, kill the level 4s
CAPTURE:FairyFeatherlink When this Monster would leave the battle area, you may suspend it to prevent that r…
CAPTURE:FairyFeatherlink When this Monster would leave the battle area, you may suspend it to prevent that r…
CAPTURE:FairyFeatherlink When this Monster would leave the battle area, you may suspend it to prevent that r…
CAPTURE:Activate FairyFeatherlink suspending
CAPTURE:Don't pay/activate
CAPTURE:FairyFeatherlink When this Monster would leave the battle area, you may suspend it to prevent that r…
CAPTURE:FairyFeatherlink When this Monster would leave the battle area, you may suspend it to prevent that r…
CAPTURE:Activate FairyFeatherlink suspending
CAPTURE:Don't pay/activate
CAPTURE:Activate FairyFeatherlink suspending
CAPTURE:Don't pay/activate
FIELD1:Four1,CS1-Sneakycat,REST,PLUG,CS3-Fairyfeatherlink Four2,CS1-Sneakycat,REST,PLUG,CS3-Fairyfeatherlink Four3,CS1-Slipperyshield,REST,PLUG,CS3-Fairyfeatherlink Five1,CS1-Phoenixdefender
FIELD2:
TRASH1:
TRASH2:CS2-Quickslash
GAME:P2 T3 MAIN 3
